govcd/lbvirtualserver_test.go

```diff
@@ -0,0 +1,184 @@
+// +build lb lbVirtualServer functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+// Test_LBVirtualServer tests CRUD methods for load balancer virtual server.
+// With help of function buildTestLBVirtualServerPrereqs such prerequisite components are created:
+// service monitor, server pool, application profile and application rule.
+// The following things are tested if prerequisites are met:
+// Creation of load balancer virtual server
+// Read load balancer virtual server by both Id and Name (virtual server name must be unique in single edge gateway)
+// Update - change a single field and compare that configuration and result objects are deeply equal
+// Update - try and fail to update without mandatory field
+// Delete
```

Comment:
Nit: I'd add numbered list here, so it's easier to notice when scanning the code.

Reply:
Added